network/peers/connections/mock/mock_id_service.go

```diff
@@ -35,3 +35,8 @@ func (m IDService) Close() error {
 	//TODO implement me
 	panic("implement me")
 }
+
+func (m IDService) Start() {
```

Comment:
don't this API change mean we should Start() IDService? did we check that it works without?

Reply:
> don't this API change mean we should Start() IDService? did we check that it works without?

correct, good catch